ftp/src/main/scala/org/apache/pekko/stream/connectors/ftp/javadsl/FtpApi.scala

```diff
@@ -573,6 +573,6 @@ object Sftp extends SftpApi {
    */
   def create(customSshClient: SSHClient): SftpApi =
     new SftpApi {
-      override val sshClient: SSHClient = customSshClient
```

Comment:
Why is this a `def` now? This could have a performance impact because it would get evaluated each time

Reply:
* more scala3 compiler issues
* the super class has `def sshClient(): SSHClient` 
* the `()` seem to be the reason that the override can't make it a `val`
* `customSshClient` is a constant as far as the anonymous class `new SftpApi {` is concerned
   * so there will be no recalculation on each call to `def sshClient(): SSHClient`
   * one option would be to make it `final def sshClient(): SSHClient` -- but with anonymous classes, you would assume everything is implied to be final anyway

Reply:
Okay all good then, approving